content/docs/CLI/build/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: build
+date: 2022-07-15
+chapter: false
+weight: 40
+bookCollapseSection: true
+---
```

Comment:
Could you please put some content here, ideally links to the subsections as one can actually navigate here and sees a blank page, i.e. http://localhost:1313/docs/cli/build/

Reply:
K